templates/store/snap-details/_distro-instructions-for-snap-support.html

```diff
@@ -129,5 +129,15 @@ <h4 class="p-media-object__title p-heading--five u-no-margin--bottom">Ubuntu</h4
         </span>
       </a>
     </div>
+    <div class="col-3 col-medium-3">
+      <a class="p-media-object" href="/install/{{package_name}}/raspbian">
+        <span class="p-media-object__image">
+          <img src="https://assets.ubuntu.com/v1/a8a6238c-RGB+RPi.svg" width="48" height="48" alt="">
+        </span>
+        <span class="p-media-object__details">
+          <h4 class="p-media-object__title p-heading--five u-no-margin--bottom">Raspberry Pi</h4>
```

Comment:
```suggestion
          <h4 class="p-media-object__title p-heading--5 u-no-margin--bottom">Raspberry Pi</h4>
```
just to be future proof

Reply:
Nice thanks - have updated all headings